assets/app/views/modals/process-template.html

```diff
@@ -0,0 +1,21 @@
+<div class="modal-resource-action">
+  <div class="modal-body">
+    <h1>You are adding a Template</h1>
+    <p>Please select type of actions you want to do with your defined Template:</p>
+
+    <div>
+      <input type="checkbox" ng-model="templateOptions.process"/>
+      <strong>Process the template</strong>
```

Comment:
Wrap the input and text in a `label` element for accessibility (and so I can click the label text to toggle the checbox).

```html
<label>
  <input type="checkbox" ...>
  Process the template
</label>
```